### ISSUE TITLE
Warn when merging page objects while using createPageObjects()

### ISSUE THREAD

**Issue by LukasHeimann** [SAP/ui5-uiveri5#223]
Hi everyone,

in my project, we have multiple teams working on a large end-to-end test with UIVeri5. Page Objects are created by different teams across different repositories. Today we found a bug that originated because two teams tried to create a (different) page object with the same name, causing code from one of the teams to not be taken. I had a brief look into the implementation, and it seems to be as expected: https://github.com/SAP/ui5-uiveri5/blob/master/src/pageObjectFactory.js. When the name is the same, the definitions will be merged and possibly overwritten.

Would it be possible to log a warning when such a thing happens? I don't know if it is a common pattern to split the page object definition into multiple `createPageObjects` calls, but for us it was simply a bug that was hard to track down.

If required, I can also put forward a small PR myself -- this should not be more then a few lines of code.

Thank you for considering this new feature!
Kind regards
Lukas

**Comment by maximnaidenov** [comment]
Hi Lukas,
I would say this is more of a feature than a bug. Adding warning is fine, if you prepare a PR I would be glad to merge.

**Comment by LukasHeimann** [comment]
Hi Maxim,

Thank you for your quick response. I've opened https://github.com/SAP/ui5-uiveri5/pull/224.

I found that there is no such thing as `.warn()` on your logger's implementation. Using it would be way better then `.info()`, to which I switched now. Adding warnings would need a bit more thinking, though. I guess you would want to show them by default like INFO and ERROR. But, you'd maybe also want to disable them. Either way, this would be changing the log level definitions.

I'd be happy to add `.warn()` to the logger in the same way that INFO and ERROR work, just with a different prefix. But it is up to you if I do this.
